micro-rdk/src/esp32/pwm.rs

```diff
@@ -166,6 +165,50 @@ impl From<PwmChannel> for usize {
     }
 }
 
+pub enum LedcTimerOption<'a> {
+    Timer0(LedcTimerDriver<'a, TIMER0>),
```

Comment:
There is a couple of place where `LedcTimerDriver` is returned or used would need to add the new generic param